sway/commands/output/background.c

```diff
@@ -1,4 +1,4 @@
-#define _XOPEN_SOURCE 500
+#define _XOPEN_SOURCE 600
```

Comment:
I'd rather replace these with _POSIX_C_SOURCE as appropriate

Reply:
It's done.